api/src/formatters/php.ts

```diff
@@ -0,0 +1,43 @@
+import { Exporter, IntermediateTranslationFormat, Parser } from '../domain/formatters';
+import { parse } from 'php-array-parser';
+import { jsonNestedParser } from './jsonnested';
+
+export const phpParser: Parser = async (data: string) => {
+  let php = data.substr(data.indexOf('['));
+  php = php.substr(0, php.lastIndexOf(';') > 0 ? php.lastIndexOf(';') : php.length);
+  return jsonNestedParser(JSON.stringify(parse(php)));
+};
+
+export const phpExporter: Exporter = async (data: IntermediateTranslationFormat) => {
```

Comment:
Wouldn't you need to escape the term and translations in case there is quote in there?

For example in:

```php
[  "term with \" quote" => "translation with \" quote"  ]
```